src/yadism/coefficient_functions/light/g4_nc.py

```diff
@@ -0,0 +1,51 @@
+from ..partonic_channel import RSL
+from . import nlo
+from . import partonic_channel as pc
+
+
+class NonSinglet(pc.LightBase):
+    @staticmethod
+    def LO():
+        """
+        |ref| APFEL
```

Comment:
You can actually cite Eq. (4) of hep-ph/9412255. In this case it 'll be more apparent that it is indeed a delta term.